recipes/bloaty/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+from conans import CMake, ConanFile, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class BloatyConan(ConanFile):
+    name = "bloaty"
+    description = """Bloaty McBloatface: a size profiler for binaries"""
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/bloaty"
+    license = "Apache-2.0"
+    topics = ("conan", "bloaty", "profiler", "size")
+
+    settings = "os", "arch", "compiler", "build_type"
+
+    requires = (
+        "capstone/4.0.2",
+        "protobuf/3.17.1",
+        "pkgconf/1.7.4",
+        "zlib/1.2.11",
+        "re2/20210601",
+        "abseil/20210324.2",
+    )
+    exports_sources = "CMakeLists.txt", "patches/**"
+    generators = "cmake", "cmake_find_package", "pkg_config"
+
+    _source_subfolder = "source_subfolder"
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, "17")
+
+    def source(self):
```

Comment:
If I understand correctly, then `bloaty` is binary only and supposed to be used as a build requirement.
Then it does not matter what compiler it is built with.
```suggestion
            tools.check_min_cppstd(self, "17")

    def package_id(self):
        del self.info.settings.compiler

    def source(self):
```